tools/importTool/src/main/scala/BytesAccumulatorActor.scala

```diff
@@ -0,0 +1,33 @@
+import java.nio.file.{Files, Paths}
+
+import akka.actor._
+import akka.pattern.ask
+import akka.util.Timeout
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+
+
+case class Message(msg: Int)
```

Comment:
please rename `msg: Int` to `value: Int`

Reply:
done